app/features/settings/settings.tsx

```diff
@@ -92,6 +94,12 @@ export default function Settings() {
     await shareContent(data);
   };
 
+  const handleSignOut = async () => {
+    setLoading(true);
+    await signOut({ redirectTo: '/signup' });
```

Comment:
passing `{ redirectTo: '/signup' }` solves the problem where our logout would add redirectTo=/settings query param and on next login we'd be taken to settings instead of home page. this was pretty annoying to me